ci-operator/step-registry/gather/must-gather-network/OWNERS

```diff
@@ -0,0 +1,11 @@
+approvers:
```

Comment:
@pecameron let's add some people from the networking team here since this is networking-related.

Reply:
OK